e2e/cli_test.go

```diff
@@ -6,13 +6,39 @@ import (
 	"bytes"
 	"encoding/json"
 	"io"
+	"log"
 	"os"
 	"os/exec"
+	"path/filepath"
 	"testing"
 
 	"github.com/google/go-cmp/cmp"
+	"gopkg.in/yaml.v3"
+
+	"github.com/styrainc/regal/pkg/config"
+	"github.com/styrainc/regal/pkg/report"
 )
 
+func readProvidedConfig(t *testing.T) config.Config {
+	t.Helper()
+
+	cwd := must(os.Getwd)
+
+	bs, err := os.ReadFile(filepath.Join(cwd, "..", "bundle", "regal", "config", "provided", "data.yaml"))
+	if err != nil {
+		log.Fatal(err)
+	}
```

Comment:
I think I'd have gone with t.Fatalf, since it's already a helper function.

Reply:
See cce360e0cde331dadcb3d3f9ae13c0c6f27db472